pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -257,3 +272,55 @@ func (c *cephStatusChecker) conditionMessageReason(condition cephv1.ConditionTyp
 
 	return condition, reason, message
 }
+
+// forceDeleteStuckPodsOnNotReadyNodes lists all the nodes that are in NotReady state and
+// gets all the pods on the failed node and force delete the pods stuck in terminating state.
+func forceDeleteStuckPodsOnNotReadyNodes(context *clusterd.Context, namespace, clusterName string) error {
```

Comment:
This does not delete all stuck pods but only stuck pods for ceph-csi. Suggest renaming to `forceDeleteStuckCephCSIPodsOnNotReadyNodes`

Reply:
It deletes stuck pods related to Rook, so naming it `forceDeleteStuckRookPodsOnNotReadyNodes`